subsys/bluetooth/mesh/Kconfig

```diff
@@ -734,6 +734,89 @@ config BT_MESH_FRIEND_SEG_RX
 
 endif # BT_MESH_FRIEND
 
+config BT_MESH_CFG_SRV_OPT
```

Comment:
Move this to a separate file and make it a sub-menu of a CFG_SRV instead:

In this file:
```
rsource "Kconfig.cfg_srv"
```
In Kconfig.cfg_srv:
```
menu "Configuration Server features"

config BT_MESH_CFG_SRV_DEV_COMP

# and so on...

endmenu
```